scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -564,7 +564,8 @@ def _update_test_status(self):
     def _parse_report_file(self, report):
         tree = ET.parse(report)
         root = tree.getroot()
-        if elem_ts := root.find('testsuite'):
+        elem_ts = root.find('testsuite')
+        if elem_ts is not None:
```

Comment:
What was wrong with the previous approach?

Reply:
Python 3.12 (at least) reports a deprecation warning about relying on "truthiness".
see 7933d2f43547531d960d002e89b26eae98fb8b62’s commit message for a quote of that message.
I am not well versed enough in python to figure where this comes from though :(

Reply:
I guess `if (elem_ts := root.find('testsuite')) is not None:` might also work ?